src/Symfony/Bundle/SecurityBundle/Tests/Functional/Bundle/SimpleLoginBundle/SimpleLoginBundle.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Tests\Functional\Bundle\SimpleLoginBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+class SimpleLoginBundle extends Bundle
+{
+    public function getParent()
+    {
+        return 'FormLoginBundle';
+    }
+}
```

Comment:
Could we just use the `FormLoginBundle` directly in the tests? Then we could remove this file and the new `after_login.html.twig`. That bundle already gives us a functional `/profile` URL, which is all we really need I think.